plugins/CoreAdminHome/Menu.php

```diff
@@ -66,7 +66,8 @@ public function configureTopMenu(MenuTop $menu)
                     $menu->addItem('CoreAdminHome_WhatIsNew', null, 'javascript:', 990,
                         Piwik::translate('CoreAdminHome_WhatIsNewTooltip'),
                         $icon, "Piwik_Popover.createPopupAndLoadUrl('module=CoreAdminHome&action=whatIsNew', '".
-                        addslashes(Piwik::translate('CoreAdminHome_WhatIsNewTooltip'))."','what-is-new-popup')");
+                        addslashes(Piwik::translate('CoreAdminHome_WhatIsNewTooltip'))."','what-is-new-popup')",
+                        null, null, $userChanges->getNewChangesCount());
```

Comment:
That'd be a lot nicer and make the properties more discoverable and easier to document. It look like `addItem` is part of the [API](https://developer.matomo.org/api-reference/Piwik/Menu/MenuAbstract) so it might not be easy so change. One for the future! :slightly_smiling_face: